stripped/colabs/Analiza_szeregow_czasowych_dot_roslinnosci_stripped.py

```diff
@@ -0,0 +1,180 @@
+"""
+<a href="https://colab.research.google.com/github/PiotrMaciejKowalski/BigData2024Project/blob/Analiza-szeregow-czasowych-dot-roslinnosci/colabs/Analiza_szereg%C3%B3w_czasowych_dot_ro%C5%9Blinno%C5%9Bci.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>
+"""
+
+"""
+# Wczytywanie danych w sparku
+"""
+
+"""
+Utworzenie środowiska pyspark do obliczeń:
+"""
+
+!apt-get install openjdk-8-jdk-headless -qq > /dev/null
+!wget -q dlcdn.apache.org/spark/spark-3.5.0/spark-3.5.0-bin-hadoop3.tgz
+!tar xf spark-3.5.0-bin-hadoop3.tgz
+!pip install -q findspark
+
+
+import os
+os.environ["JAVA_HOME"] = "/usr/lib/jvm/java-8-openjdk-amd64"
+os.environ["SPARK_HOME"] = "/content/spark-3.5.0-bin-hadoop3"
+import findspark
+findspark.init()
+from pyspark.sql import SparkSession
+from google.colab import drive
+from pyspark.sql.types import IntegerType, FloatType, StringType, StructType
+import pandas as pd
+
+
+"""
+Utowrzenie sesji:
+"""
+
+spark = SparkSession.builder\
+        .master("local")\
+        .appName("Colab")\
+        .config('spark.ui.port', '4050')\
+        .getOrCreate()
+
+
+"""
+Połączenie z dyskiem:
+"""
+
+drive.mount('/content/drive')
+
+
+"""
+Wczytanie danych NASA znajdujących się na dysku w sparku:
+"""
+
+# Wczytanie zbioru sampled w celu pobrania nazw kolumn
+sampled = pd.read_csv('/content/drive/MyDrive/BigMess/NASA/sampled_NASA_200k.csv')
+
+# Utworzenie schematu określającego typ zmiennych
+schemat = StructType()
```

Comment:
mamy dwujęzyczność w zakresie nazywania zmiennych. To zła praktyka. Przejdźmy na angielski.